src/views/userOp/RecentUserOps.tsx

```diff
@@ -0,0 +1,615 @@
+import Footer from '@/components/globals/footer/Footer';
+import Navbar from '@/components/globals/navbar/Navbar';
+import React, { useEffect, useState } from 'react';
+import { getUserOp, GetUserOpsHash } from '@/components/common/apiCalls/jiffyApis';
+import { Breadcrumbs, Link } from '@mui/material';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import CopyButton from '@/components/common/copy_button/CopyButton';
+import Caption from '@/components/common/table/Caption';
+import IconText from '@/components/common/IconText';
+import Chip from '@/components/common/chip/Chip';
+import sx from './usertable.module.sass';
+import { useRouter } from 'next/router';
+export const BUTTON_LIST = [
+    {
+        name: 'Default View',
+        key: 'Default View',
+    },
+    {
+        name: 'UTF-8',
+        key: 'UTF-8',
+    },
+    {
+        name: 'Original',
+        key: 'Original',
+    },
+];
+
+function RecentUserOps() {
+    const router = useRouter();
+    const [open, setOpen] = useState(false);
+    const { hash } = router?.query;
+    const [selectedColor, setSelectedColor] = useState('#1976D2');
+    const [useOpsData, setuserOpsData] = useState<GetUserOpsHash[]>();
+    function shortenString(str: string) {
+        if (str?.length <= 10) {
+            return str;
+        }
+        const firstChars = str?.slice(0, 6);
+        const lastChars = str?.slice(-4);
+
+        return `${firstChars}...${lastChars}`;
+    }
+    useEffect(() => {
+        const refreshUserOpsTable = async (name: string) => {
+            const userops = await getUserOp(name || '0x43fe1ef830cbc6447ca8a740963099fe7fb6b137ac7768aa9c8f5913aaf8f91b');
+            setuserOpsData(userops);
+        };
+        refreshUserOpsTable(hash as string);
+    }, [hash]);
+
+    function getSymbol(network: string): string {
```

Comment:
Please reference these function from the utils file. Don't duplicate them again.